portfolio/src/main/webapp/index.html

```diff
@@ -110,6 +108,14 @@ <h3 id="contact_me">Contact Me</h3>
                placeholder="example@gmail.com"><br>
         <input class="button" type="submit" value="Submit">
       </form>
+      <p><b>Comment History</b></p>
+      <label for="maxN">Maximum number of entries to show in comment history:</label>
+      <input type="number" id="maxN" name="maxN" onchange="addComments()"
+             min="0" max="" value="">
+      <p>(The latest comments are displayed on the top.)</p>
```

Comment:
nit: "The latest comments are displayed at the top" or "The latest comments are displayed on top"